resources/js/processes-catalogue/components/ProcessesCatalogue.vue

```diff
@@ -15,6 +15,7 @@
           preicon="fas fa-play-circle"
           class="mt-3"
           show-bookmark="true"
+          :category-count="categoryCount"
```

Comment:
Please use the permission in order to see the new option, this is not necesary

Reply:
This is a necessary Prop to send number of categories to Create Process modal component. Without this parameter, we can't create new Processes.